packages/components/nodes/chatmodels/AWSBedrock/AWSChatBedrock.ts

```diff
@@ -138,7 +138,7 @@ class AWSChatBedrock_ChatModels implements INode {
         const iModel = nodeData.inputs?.model as string
         const iTemperature = nodeData.inputs?.temperature as string
         const iMax_tokens_to_sample = nodeData.inputs?.max_tokens_to_sample as string
-        const cache = nodeData.inputs?.llmCache as BaseCache
+        const cache = nodeData.inputs?.cache as BaseCache
```

Comment:
One might need to delete the nodes from saved flows and recreate them for this change to be seen.

Reply:
yes exactly